docs/index.html

```diff
@@ -202,7 +202,7 @@ <h2>Interactive Demo</h2>
   <div class="container">
     <h2>HPG 2024 Talk</h2>
     <hr>
-    <iframe width="100%" style="aspect-ratio: 16/9;" frameBorder="0" src="https://www.youtube.com/embed/qPmwbp7A3BQ?start=20666" allowfullscreen>
+    <iframe width="100%" style="aspect-ratio: 16/9;" frameBorder="0" src="https://www.youtube.com/embed/UrnnCKYw6aA" allowfullscreen>
```

Comment:
Shouldn't the link in the "video" button also be updated?

Reply:
Oops, done!